detection-rules/impersonation_adobe_suspicious_language_link.yml

```
@@ -65,19 +75,29 @@ source: |
           or strings.istarts_with(subject.subject, "TR:")
           or strings.istarts_with(subject.subject, "FWD:")
           or regex.imatch(subject.subject,
-                        '(\[[^\]]+\]\s?){0,3}(re|fwd?|automat.*)\s?:.*'
+                        '^\[?/{0,2}(EXT|EXTERNAL)\]?/{0,2}[: ]\s*(RE|FWD?|FW|AW|TR|ODG|答复):.*'
         )
         )
       )
     )
     or length(headers.references) == 0
   )
   and (
-    not profile.by_sender().solicited
-    or (
-      profile.by_sender().any_messages_malicious_or_spam
-      and not profile.by_sender().any_false_positives
+    (
+      headers.auth_summary.spf.pass
+      and headers.auth_summary.dmarc.pass
+      and (
+        not profile.by_sender().solicited
+        or profile.by_sender().any_messages_malicious_or_spam
+        or profile.by_sender_email().days_since.last_contact > 14
+      )
+      and not profile.by_sender().any_messages_benign
+      and not sender.email.domain.root_domain in ("adobe-events.com", "frame.io")
     )
+    or not headers.auth_summary.spf.pass
+    or headers.auth_summary.spf.pass is null
+    or not headers.auth_summary.dmarc.pass
+    or headers.auth_summary.dmarc.pass is null
```

thoughts on coalesce? 
```suggestion
    or not coalesce(headers.auth_summary.spf.pass, false)
    or not coalesce(headers.auth_summary.dmarc.pass, false)
```